wire-ios-sync-engine/Source/UserSession/ZMUserSession.swift

```diff
@@ -297,6 +297,14 @@ public class ZMUserSession: NSObject {
         return EnrollE2eICertificateUseCase(e2eiRepository: e2eiRepository)
     }()
 
+    public lazy var getIsE2eIdentityEnabled: GetIsE2EIdentityEnabledUsecaseProtocol =  {
```

Comment:
Why do we need `getIsE2eIdentityEnabled` property and a separate UseCase for this property?

Reply:
Each use case should only fetch one result. That's how use cases are supposed to work